src/components/App/App.jsx

```diff
@@ -35,18 +37,20 @@ const App = () => {
 
   return (
     <>
-      <GlobalStyle />
-      <Layout handleSearch={handleSearch} handleKeyPress={handleKeyPress} search={search}>
-        {page.home ? (
-          <Home handleSelectVideo={handleSelectVideo} videos={videos} />
-        ) : (
-          <DetailsPage
-            video={page.video}
-            handleSelectVideo={handleSelectVideo}
-            relatedVideos={videos.items}
-          />
-        )}
-      </Layout>
+      <ThemeProvider theme={theme}>
+        <GlobalStyle />
+        <Layout
+          handleSearch={handleSearch}
+          handleKeyPress={handleKeyPress}
+          search={search}
+        >
+          {page.home ? (
```

Comment:
with the previous comment this would get turned into

`{ currentPage === PAGES.home ? [...]`